utils/storybook/CanvasProviderDecorator.tsx

```diff
@@ -16,6 +17,13 @@ export default makeDecorator({
   wrapper: (storyFn, context, {parameters = {}}) => {
     const theme: PartialEmotionCanvasTheme = {
       canvas: object(label, parameters.theme || defaultCanvasTheme),
+      ABTest: {
+        secondaryOutlineButton: {
+          main: 'orange', //colors.licorice300,
```

Comment:
I made this super obvious, but commented the intended colors to be used in the ABtest. We can switch to these if preferred, but I was worried the snapshot tests wouldn't catch the small color changes.

Reply:
Using Storybook doesn't catch the issue where `ABTest` might not be defined on the `theme` object, but because this wraps all our stories, we wouldn't notice that issue. Cypress wouldn't catch this either because Cypress runs in the same context as Storybook.

I think a better place to define this `ABTest` theme override would be in a new `ABTest` story that wraps in a new `<CanvasProvider theme={theme}>` provider. This way the normal `OutlineButton` is proven to work without the override both visually and functionally. The new story can ensure the `ABTest` theme override works correctly both visually and functionally.

The theme override of this wrapper worries me that we might run into issues later. It is nice that unit test catch these errors, but this file make it less obvious there's an issue.